### PR TITLE
Update Profiling Notebook to "Best Practices" State

### PR DESCRIPTION
**Pull request by digitalsimboja**
This fix will ensure that  Profiling Notebook is updated to meet the "Best Practices" State

#111 

## Description

Addresses issue #xxx and aims to advance it to

### CODE REVIEW

**Comment by digitalsimboja** [comment]
@OriolAbril 

Kindly review

Thanks

File: examples/pymc3_howto/profiling.ipynb [hunk 1]
```diff
@@ -31,16 +31,6 @@
     "print(f\"Running on PyMC3 v{pm.__version__}\")"
```
Comment (OriolAbril) [inline examples/pymc3_howto/profiling.ipynb:31]: <p>Should be <code>theano,xarray</code>.  And you need to rerun the notebook for the changes to be reflected here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/157/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

**Comment by digitalsimboja** [comment]
Done,

Please take a look again.

Thanks and regards

**Comment by OriolAbril** [comment]
It looks like you did "restart and **clear** all" instead of "restart and **run** all"

**Comment by digitalsimboja** [comment]
I ran 'restart and run all'. Anytime I run this, it clears some data on the
original file.
I wonder why

**Comment by OriolAbril** [comment]
Have you tried executing the cells manually? 

The cells are clearly not executed, they have no execution count (the number shown as `[1]` on the left of the cell) and have no output at all, not the expected output nor an error message. If they had been executed all this would be there, maybe there would be errors, but something would be there as output.

**Comment by fonnesbeck** [comment]
I'm getting some invalid JSON errors both in ReviewNB and locally.

**Comment by OriolAbril** [comment]
@digitalsimboja closing the PR so someone else can work on this notebook. Do reopen if you still plan to work on it.

**Comment by digitalsimboja** [comment]
Okay please go ahead as I have been

**Pull request closed without merging**